internal/namespaces/instance/v1/testdata/test-server-delete-only-local-volumes.stdout.golden

```diff
@@ -0,0 +1 @@
+✅ Seleted the following resources as well: ip 163.172.146.186, volume bcc4897e-5e17-4b5e-a392-347e631eb2e0 (l_ssd).
```

Comment:
should use an assert instead of a golden

Reply:
goldens make more sense now